lib/services.js

```diff
@@ -90,11 +91,27 @@ module.exports = function generateServices(app, options) {
     { encoding: 'utf-8' }
   );
 
+  var commonModelPrefix = 'LoopBack';
+  if (options.namespaceCommonModels) {
+    commonModelPrefix = options.ngModuleName + options.namespaceDelimiter;
+    if (options.namespaceDelimiter === '.') {
+      return 'throw new Error("Unsupported delimiter "." for ' +
```

Comment:
I believe I was copying how getPropertyOfFirstEndpoint was working, as that was throwing the exception during the render (through the ejs helper) which from memory had the effect of writing it to the page, but I agree throwing a real exception is much better and it is easier to test. Done.